tests/cargo-kani/itoa_dep/src/main.rs

```diff
@@ -20,7 +20,7 @@ fn check_itoa<T: kani::Arbitrary + Integer + std::fmt::Display>() {
 #[kani::proof]
 #[kani::unwind(10)]
 fn check_signed() {
-    check_itoa::<i8>();
+    check_itoa::<u8>();
```

Comment:
Can we instead comment out this harness (referring to #2576) and create another harness `check_unsigned` for the `u8` type?

Reply:
Sure.